src/providers/MeetingProvider/types.ts

```diff
@@ -20,11 +19,24 @@ export enum DevicePermissionStatus {
 }
 
 export interface MeetingJoinData {
-  meetingInfo: any;
-  attendeeInfo: any;
-  deviceLabels?: DeviceLabels | DeviceLabelTrigger;
-  eventReporter?: EventReporter;
-  meetingManagerConfig?: MeetingManagerConfig;
+  meetingSessionConfiguration: MeetingSessionConfiguration;
+  options?: {
+    deviceLabels?: DeviceLabels | DeviceLabelTrigger;
+    eventReporter?: EventReporter | undefined;
+    logLevel?: LogLevel;
+    postLoggerConfig?: PostLoggerConfig | undefined;
+    logger?: Logger;
+    enableWebAudio?: boolean;
+    activeSpeakerPolicy?: ActiveSpeakerPolicy;
+  };
+}
+
+export interface MeetingManagerJoinOptions {
```

Comment:
Sorry, could you please help on where do we use this?

Reply:
It is used here: https://github.com/aws/amazon-chime-sdk-component-library-react/pull/751/files#diff-6e7e64f1faf69cc598827e0ff6e12ed9d76d6984f353ebda6d726571ec39ae4fR214.

I was initially trying to use `MeetingManagerJoinData["options"]` but that did not work because I think TS was having issue with optionality of the props.